common/lib/capa/capa/templates/codeinput.html

```diff
@@ -1,9 +1,11 @@
 <%! from django.utils.translation import ugettext as _ %>
 <section id="textbox_${id}" class="capa_inputtype textbox cminput">
-  <textarea rows="${rows}" cols="${cols}" name="input_${id}" 
-            aria-label="${_("{programming_language} editor").format(programming_language=mode)}" 
```

Comment:
@cptvitamin I have removed `aria-label` from the `textarea`. Do you think its OK?